packages/sindarian-i18n-cli/src/reporter.ts

```
@@ -0,0 +1,114 @@
+  const useColors = options?.colors ?? process.stdout.isTTY === true
+
+  const grouped = new Map<string, ValidationIssue[]>()
+  for (const issue of result.issues) {
+    const list = grouped.get(issue.file) ?? []
+    list.push(issue)
+    grouped.set(issue.file, list)
+  }
+
+  const lines: string[] = []
+
+  for (const file of [...grouped.keys()].sort()) {
```

ESLint

[error] Use grouped.keys().toSorted() instead of copying and sorting

(e18e/prefer-array-to-sorted)